.travis.yml

```diff
@@ -1,14 +1,17 @@
 language: android
+jdk: oraclejdk8
 android:
   components:
-    - build-tools-19.1.0
-    - android-19
+    - platform-tools
+    - tools
+    - build-tools-25.0.1
+    - android-25
+node_js:
```

Comment:
No. They are, I think, mutually exclusive. If you set it to `node_js` then you have to manually install all the Android stuff. If you set it to `android` you need to manually manage node versions. You can use an environment matrix if we are dead set on testing on different nodes.